frontend/server/src/Controllers/Course.php

```diff
@@ -1640,10 +1904,12 @@ public static function getIntroDetails(\OmegaUp\Request $r): array {
                 $course,
                 $group
             );
-            $isFirstTimeAccess =
-                is_null($sharingInformation['share_user_information']);
-            $shouldShowAcceptTeacher =
-                is_null($sharingInformation['accept_teacher']);
+            $isFirstTimeAccess = empty(
```

Comment:
```suggestion
            [
                'share_user_information' => $hasSharedUserInformation,
                'accept_teacher' => $hasAcceptedTeacher,
            ] = \OmegaUp\DAO\Courses::getSharingInformation( 
```

y en vez de regresar `null|bool`, que regrese `bool` para hacerte la vida más fácil aquí porque no hay necesidad de distinguir entre `false` y `null`.

Reply:
Listo